tests/test_main.py

```diff
@@ -30,7 +30,15 @@ def setUp(self):
         }
 
         # Mock Fedmsg Message
-        self.mock_message = {"msg_id": "mock_id", "msg": {"issue": "mock_issue"}}
+        self.mock_message_body = {"issue": "mock_issue"}
+        self.old_style_mock_message = {
+            "msg_id": "mock_id",
+            "msg": self.mock_message_body,
+        }
+        self.new_style_mock_message = {
+            "msg_id": "mock_id",
+            "msg": self.mock_message_body,
+        }
```

Comment:
The key in the new style message (line 40) is supposed to be `body`, right?

Reply:
You're right! Hasty, hasty.